netlogo-gui/src/main/lab/gui/RunOptionsDialog.scala

```diff
@@ -32,16 +36,20 @@ class RunOptionsDialog(parent: java.awt.Dialog,
   class EditableRunOptions extends Editable {
     var spreadsheet = Prefs.spreadsheet
     var table = Prefs.table
+    var updateView = Prefs.updateView
+    var updatePlotsAndMonitor= Prefs.updatePlotsAndMonitor
     var threadCount = Runtime.getRuntime.availableProcessors
     val classDisplayName = "Run options"
     val propertySet =
       List(
         Property("spreadsheet", Property.Boolean, "Spreadsheet output"),
         Property("table", Property.Boolean, "Table output"),
+        Property("updateView",Property.Boolean,"Update View"),
+        Property("updatePlotsAndMonitor",Property.Boolean,"Update Plots and Monitor"),
```

Comment:
The user facing strings should only have the first letter capitalized and (more importantly) "monitor" should be plural, i.e.: `"Update view"` and `"Update plots and monitors"`. "Monitor" should be plural in property and variable names as well.